source/htslib/kstring.d

```diff
@@ -199,6 +199,13 @@ void ks_free(kstring_t* s)
     }
 }
 
+kstring_t* ks_dup(kstring_t* s) {
```

Comment:
Interestingly is not a part of kstring.h or kstring.c ; will this make it more difficult to keep your file in sync with upstream?

Reply:
Since we have the diff system, probably not. Though I should update the diffs considering I have new changes that need to carry forward. 

Though I also thought that `kstring` already had a `dup` function and was surprised to find that it didn't.